File: Project/Android/app/src/main/java/com/example/recetapp/model/recipe/RecipeByIngredients.kt
package com.example.recetapp.model.recipe

import com.example.recetapp.model.recipe.ingredients.ExtendedIngredient

data class RecipeByIngredients(
    val id: Int,
    val title: String,
    val image: String,
    val usedIngredientCount: Int,
    val unusedIngredientCount: Int,
    val missedIngredientCount: Int,
    val missedIngredients: List<ExtendedIngredient>,
    val unusedIngredients: List<ExtendedIngredient>,
    val usedIngredients: List<ExtendedIngredient>
)

File: Project/iOS/RecetApp/RecetApp/Model/RecipeByIngredients.swift
//
//  RecipeByIngredients.swift
//  RecetApp
//
//  Created by Andrea García Ruiz on 22/03/23.
//

import Foundation

// MARK: - WelcomeElement
struct RecipeByIngredients: Codable {
    let id: Int
    let image: String
    let missedIngredientCount: Int
    let missedIngredients: [ExtendedIngredient]
    let title: String
    let unusedIngredients: [ExtendedIngredient]
    let usedIngredientCount: Int
    let usedIngredients: [ExtendedIngredient]
}
